src/StudentOrganizer.Infrastructure/Extentions/TimeExtentions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StudentOrganizer.Infrastructure.Extentions
+{
+    public static class TimeExtentions
+    {
+        public static long ToTimestamp(this DateTime dateTime)
+        {
+            var epoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
Można zrobić taki hack
((DateTimeOffset)dateTime).ToUnixTimeMilliseconds()

Reply:
https://stackoverflow.com/questions/17632584/how-to-get-the-unix-timestamp-in-c-sharp stąd brałem xd

Reply:
Ładniejsza opcja z tą metodą ToUnixTimeMilliseconds imo